### PR TITLE
Fix kubeadm command related typos, punctuation and capitalization

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:
This PR deals with a numerous fixes of small punctuation, capitalization and wording typos in the kubeadm/app/cmd namespace. It cleans the output shown to the user in stdout.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
/approve no-issue

**Special notes for your reviewer**:
I can easily amend, remove or squash certain changes, so if you think there is a need for that - please, let me know.

Lubomir I. Ivanov (VMWare)

### CODE REVIEW

**Comment by ncdc** [comment]
I'll let the others review, but I don't think this needs a release note.
/unassign

**Comment by deads2k** [comment]
Doesn't need a release note.  Does need a squash.

File: cmd/kubeadm/app/cmd/completion.go [hunk 1]
```diff
@@ -113,14 +113,14 @@ func NewCmdCompletion(out io.Writer, boilerPlate string) *cobra.Command {
 // RunCompletion checks given arguments and executes command
 func RunCompletion(out io.Writer, boilerPlate string, cmd *cobra.Command, args []string) error {
 	if len(args) == 0 {
-		return fmt.Errorf("shell not specified")
+		return fmt.Errorf("Shell not specified.")
```
Comment (deads2k) [inline cmd/kubeadm/app/cmd/completion.go]: errors always lead with lower case.  They don't need a period at the end.

File: cmd/kubeadm/app/cmd/init.go [hunk 2]
```diff
@@ -145,35 +145,35 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
...
 	)
 	flagSet.StringVar(
 		&cfg.Networking.PodSubnet, "pod-network-cidr", cfg.Networking.PodSubnet,
-		"Specify range of IP addresses for the pod network; if set, the control plane will automatically allocate CIDRs for every node",
+		"Specify range of IP addresses for the pod network; if set, the control plane will automatically allocate CIDRs for every node.",
 	)
 	flagSet.StringVar(
 		&cfg.Networking.DNSDomain, "service-dns-domain", cfg.Networking.DNSDomain,
-		`Use alternative domain for services, e.g. "myorg.internal"`,
+		`Use alternative domain for services, e.g. "myorg.internal."`,
```
Comment (deads2k) [inline cmd/kubeadm/app/cmd/init.go]: Adding a dot here changed the meaning of the sentence.

File: cmd/kubeadm/app/cmd/init.go [hunk 3]
```diff
@@ -218,10 +218,10 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight,
 	if cfgPath != "" {
 		b, err := ioutil.ReadFile(cfgPath)
 		if err != nil {
-			return nil, fmt.Errorf("unable to read config from %q [%v]", cfgPath, err)
+			return nil, fmt.Errorf("Unable to read config from %q [%v]", cfgPath, err)
```
Comment (deads2k) [inline cmd/kubeadm/app/cmd/init.go]: errors are lower case

Comment (neolit123) [reply cmd/kubeadm/app/cmd/init.go]: i think i made a lot of these uppercase, do you want them all reverted to lowercase, or does this only apply to those which you commented in your review? thanks.

**Comment by neolit123** [comment]
hello @deads2k, i have made the requested changes. please, review.

**re: about not capitalizing error messages**
> errors always lead with lower case. They don't need a period at the end.

in certain cases an error message has to be composed of two sentences - e.g:
in the file `kubeadm/app/cmd/completion.go`, there is a:
`return fmt.Errorf("too many arguments. expected only the shell type")`

i have left this untouched, but it seems there is a little bit of contradiction here.

**Comment by mikedanese** [comment]
please squash commits

**Comment by neolit123** [comment]
> please squash commits
@mikedanese one commit per file changed?

**Comment by mikedanese** [comment]
One commit please.

**Comment by neolit123** [comment]
> One commit please.

alright, will re-submit in 15 minutes.

**Comment by neolit123** [comment]
@mikedanese, @deads2k

commits squashed into one; the changes are still noted in the commit message. please, review.

BTW, where can i find the TODO for kubeadm?

**Comment by deads2k** [comment]
/lgtm
/approve no-issue



> BTW, where can i find the TODO for kubeadm?

Not sure.  I'd ask @luxas

**Comment by neolit123** [comment]
erm,

```
Error lines from build-log.txt
<snip>
cmd/kubeadm/app/cmd/config.go:117:37: error strings should not be capitalized or end with punctuation or a newline
```
`hack/verify-golint.sh` does not fail locally, odd.

**Comment by neolit123** [comment]
i fixed this:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53139/pull-kubernetes-verify/53844/

can't retest, though.

**Comment by neolit123** [comment]
the tests have passed. @deads2k could you please lgtm if you feel it's finally good to go. sorry... :(

**Comment by deads2k** [comment]
Looks like some things changed.

/assign luxas

File: cmd/kubeadm/app/cmd/init.go [hunk 4]
```diff
@@ -191,22 +191,22 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.MasterConfigur
 func AddInitOtherFlags(flagSet *flag.FlagSet, cfgPath *string, skipPreFlight, skipTokenPrint, dryRun *bool) {
 	flagSet.StringVar(
 		cfgPath, "config", *cfgPath,
-		"Path to kubeadm config file (WARNING: Usage of a configuration file is experimental)",
+		"Path to kubeadm config file (WARNING: Usage of a configuration file is experimental).",
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/init.go]: You dropped the parentheses for this same message in `cmd/kubeadm/app/cmd/config.go:130`, can you drop them here too for consistency?

**Review by mattmoyer — APPROVED** [review]
Looks good to me with one small suggestion.

**Comment by mattmoyer** [comment]
/lgtm cancel

Whoops, let me give you a chance to fix that one inconsistency.

File: cmd/kubeadm/app/cmd/init.go (hunk 4 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/init.go]: will update in a minute. let me fire up my build setup...

**Comment by neolit123** [comment]
@mattmoyer done. rebased and pushed.

**Comment by neolit123** [comment]
thanks

**Pull request merged**